registry/handlers/app.go

```diff
@@ -51,6 +52,9 @@ const randomSecretSize = 32
 // defaultCheckInterval is the default time in between health checks
 const defaultCheckInterval = 10 * time.Second
 
+// context key for storing the Cloudflare True-Client-IP header
+const cfRealIPKey string = "cf-true-client-ip"
```

Comment:
Is this how the key will appear in logs? It should be consistent with the other http request level log fields which have names like `http.request.<key-name>`, eg here we would have `http.request.cf-true-client-ip`.

Here's a single log entry example with `http.request.cf-ray-id`: https://logs.internal.digitalocean.com/app/discover#/doc/default/default-2023.02.03?id=HOP5GIYBoXrB1R3vAfq1

(elasticsearch converts periods to underscores when ingesting logs, so from the point of view of future engineers trying to understand where a particular log is coming from it might make sense to use underscores instead of periods in code here)

Reply:
@waynr yeah it will appear in the logs using that key I can change that - the internal PR I raised has a sample log output on DOCR data plane